streams/src/main/java/org/apache/kafka/streams/state/internals/StoreQueryUtils.java

```diff
@@ -218,9 +224,9 @@ private static <R> QueryResult<R> runRangeQuery(
 
     @SuppressWarnings("unchecked")
     private static <R> QueryResult<R> runKeyQuery(final Query<R> query,
-                                                  final PositionBound positionBound,
-                                                  final QueryConfig config,
-                                                  final StateStore store) {
+        final PositionBound positionBound,
+        final QueryConfig config,
+        final StateStore store) {
```

Comment:
revert indention change (similar below)

Do you have some IntelliJ auto-formatting enabled? I would recommend to disable it to avoid stuff like this.